lib/renderer.js

```diff
@@ -725,39 +725,45 @@
             resultDeferred = Q.defer(),
             boundsAcquiredPromise,
             pixmapParams,
-            boundsResult,
-            scaleX = component.scaleX || component.scale || 1,
-            scaleY = component.scaleY || component.scale || 1;
+            maskBounds,
+            staticBounds,
+            canvasBounds;
+        
+        if (layer) {
+            if (layer.artboard) {
+                //pixmap from PS is automatically clipped to artboard. Apply that here so we
+                //know what the expected results are
+                staticBounds = this._getContentBounds(layer, true).intersect(layer.artboard);
+                canvasBounds = layer.artboard;
+            } else {
+                maskBounds = layer.getTotalMaskBounds();
+                staticBounds  = new Bounds(this._generator.getDeepBounds(layer));
+            }
```

Comment:
no, the `canvasBounds` is the size we need to expand the image too if it doesn't actually make it that large